applications/serial_lte_modem/src/slm_at_tcpip.c

```diff
@@ -661,8 +661,11 @@ static int do_sendto(const char *url, uint16_t port, const uint8_t *data,
 static int do_recvfrom(uint16_t length)
 {
 	int ret;
+	/* Below two are not used, just for static code analysis */
+	struct sockaddr src_addr;
```

Comment:
What's the exact complaint related to this change? There's nothing wrong with providing NULL addr pointers to the `recvfrom()` call.

Reply:
Dereference of NULL pointer is not allowed by Coverity

Reply:
> Dereference of NULL pointer is not allowed by Coverity

Well, this shouldn't be allowed by any sane programmer, the question is **who** dereferences it. Because as I said earlier, providing NULL address pointers is acceptable by `recvfrom` and it'll take care of verifying they're not NULL before using them.

Reply:
Guess so, but Coverity complains like below:
CID 111527 (#1 of 1): Explicit null dereferenced (FORWARD_NULL)
var_deref_model: Passing NULL to recvfrom, which dereferences it. [Note: The source code implementation of the function has been overridden by a builtin model.]